config/sidekiq.yml

```diff
@@ -3,8 +3,8 @@
 :concurrency:  2
 :logfile: ./log/sidekiq.json.log
 :queues:
-  - content_store_high # Deprecated, to be removed once jobs completed
-  - content_store_low # Deprecated, to be removed once jobs completed
+  - content_store_high
```

Comment:
I think you're on an outdated diff there (from when this was closer to being merged gradually: https://github.com/alphagov/publishing-api/pull/439). This one here should be clearer: https://github.com/alphagov/publishing-api/pull/444/commits/f03cba17d4f360ef3f90875a8c4d105225277581

It would be keeping the content_store_* ones until the sidekiq queue for those is empty.